client/src/rpc_config.rs

```diff
@@ -49,6 +49,14 @@ pub struct RpcLeaderScheduleConfig {
     pub commitment: Option<CommitmentConfig>,
 }
 
+#[derive(Debug, Default, Clone, PartialEq, Serialize, Deserialize)]
+#[serde(rename_all = "camelCase")]
+pub struct RpcGetVoteAccountsConfig {
+    pub vote_pubkey: Option<String>, // validator vote address, as a base-58 encoded string
```

Comment:
Breaks my heart a little to use `vote_pubkey` instead of `vote_address`, but I wanted to be consistent with the returned object.

I also only added support to filter by vote account address instead of validator identity because this method is about getting vote accounts.

Reply:
😢 yeah. I suppose we could deprecate the `x_pubkey` fields and add `x_address` fields instead